src/map/battle.cpp

```diff
@@ -1569,6 +1575,11 @@ int64 battle_calc_damage(struct block_list *src,struct block_list *bl,struct Dam
 			damage *= 4;
 #endif
 
+		if (tsc->getSCE(SC_GROUNDGRAVITY) && flag&(BF_MAGIC|BF_WEAPON) && !status_bl_has_mode(bl, MD_STATUSIMMUNE))
```

Comment:
```suggestion
		if (tsc->getSCE(SC_GROUNDGRAVITY) && flag&(BF_MAGIC|BF_WEAPON))
```
Do we really need to check this again? It cannot be started on bosses anyway.

Reply:
Agreed, the status has `BossResist` so it should not have applied to a boss type monster.